tests/SchemaAnalyzerTest.php

```diff
@@ -550,15 +549,12 @@ public function testChainedJunctionTables()
         $this->assertEquals('role_users', $fks[3]->getName());
     }
 
-    /**
-     * @expectedException \Mouf\Database\SchemaAnalyzer\SchemaAnalyzerTableNotFoundException
-     * @expectedExceptionMessage Could not find table 'rights'. Did you mean 'right'?
-     */
     public function testWringTableName()
     {
         $schemaManager = $this->getCompleteSchemaManager();
 
         $schemaAnalyzer = new SchemaAnalyzer($schemaManager);
+        $this->expectException(SchemaAnalyzerTableNotFoundException::class, "Could not find table 'rights'. Did you mean 'right'?");
```

Comment:
I may be wrong, but `"Could not find table 'rights'. Did you mean 'right'?"` should be in an `expectExceptionMessage`, no ?

```suggestion
        $this->expectException(SchemaAnalyzerTableNotFoundException::class);
        $this->expectExceptionMessage("Could not find table 'rights'. Did you mean 'right'?");
```